.scalafmt.conf

```diff
@@ -1,20 +1,27 @@
-maxColumn = 80
-
-style = IntelliJ
-danglingParentheses = true
-indentOperator = spray
+align = none
```

Comment:
Thanks for the reminder, I'll add in a comment to the Jira ticket tracking this. Other than that, I'll leave this option enabled as it is only currently impacting Scala.js projects.